src/app/saga.test.ts

```diff
@@ -584,6 +584,10 @@ describe('appInit', () => {
 
     expect(initializeSentry).toHaveBeenCalledTimes(1)
     expect(ValoraAnalytics.init).toHaveBeenCalledTimes(1)
+    // Ensure the right context is used
+    // Note: switch to mock.contexts[0] when we upgrade to jest >= 28
+    // See https://jestjs.io/docs/mock-function-api/#mockfnmockcontexts
+    expect(mocked(ValoraAnalytics.init).mock.instances[0]).toBe(ValoraAnalytics)
```

Comment:
`mock.instances` "works" but wasn't really meant for this. See this [comment](https://stackoverflow.com/questions/52027328/how-to-test-this-calling-context-with-jest#comment91033973_52031787).
But until we can update to jest >= 28, this is better than nothing.